lib/challenge.js

```diff
@@ -30,7 +35,11 @@
   // - a name
   // - an address
 
-var pet_owner;
+var pet_owner = {
+  name:'baxter',
+  address: 'change st, somerville, ma 02145',;
```

Comment:
This object literal is broken by that semicolon. Semicolons have no place in key-value pairs, which are all that is allowed directly inside an object literal. When a semicolon occurs here, then, the interpreter throws a syntax error, and does not create the object.